wikidict/lang/en/labels.py

```diff
@@ -492,19 +493,9 @@
     "women's speech": "women's speech",
     "wrt": "with respect to",
     "youth slang": "youth slang",
-}  # 487
+}  # 488
 
 label_syntaxes = {
-    "&": {
```

Comment:
A bit too fast on merge. I should have added a draft tag. Some aliases have been accidentally removed since they changed the syntax again on wk...

Reply:
Ah yes, sorry 😓
I don't know if you want  to merge your PRs or if I should do it sometimes. Well, It's not a big deal, but let me kown if you need me to do something some time :)